src/main/java/uk/gov/pay/adminusers/resources/ServiceRequestValidator.java

```diff
@@ -100,7 +100,7 @@ private void validateMerchantEmail(JsonNode payload) throws ValidationException
     }
 
     public Optional<Errors> validateSearchRequest(ServiceSearchRequest request) {
-        var allowedChars = Pattern.compile("^[0-9A-Za-z\\s]+$");
+        var allowedChars = Pattern.compile("^[0-9A-Za-z'\\-\\s]+$");
```

Comment:
While you’re here, you could make this into a constant so we don’t compile the regex every time.